INSTALL.md

```diff
@@ -0,0 +1,233 @@
+# INSTALL / BUILD instructions for Apache Airflow
+
+This is a generic installation method that requires a number of dependencies to be installed.
+
+Depending on your system you might need different prerequisites, but the following
+systems/prerequisites are known to work:
+
+Linux (Debian Bullseye and Linux Mint Debbie):
+
+      sudo apt install build-essential python3-dev libsqlite3-dev openssl \
+                 sqlite default-libmysqlclient-dev libmysqlclient-dev postgresql
+
+On Ubuntu 20.04 you may get an error of ```mariadb_config not found```
+and mysql_config not found.
+
+Install MariaDB development headers:
+
+      sudo apt-get install libmariadb-dev libmariadbclient-dev
+
+MacOS (Mojave/Catalina):
+
+      brew install sqlite mysql postgresql
+
+- [Required] Fetch the tarball and untar the source move into the directory that was untarred.
+- [Optional] Run Apache RAT (release audit tool) to validate license headers
+- RAT docs here:
+  ```https://creadur.apache.org/rat/.```
+- Requires Java and Apache Rat:
+  
+      java -jar apache-rat.jar -E ./.rat-excludes -d
+
+**[optional] Airflow pulls in quite a lot of dependencies in order
+to connect to other services. You might want to test or run Airflow
+from a virtual env to make sure those dependencies are separated
+from your system wide versions**
+
+      python3 -m venv PATH_TO_YOUR_VENV 
+      source PATH_TO_YOUR_VENV/bin/activate
+
+### [required] Building and installing by pip (preferred)
+      pip install .
+
+#### Or directly
+      python setup.py install
```

Comment:
Don’t do this please

Reply:
Can you provide with a better alternative? I'll surely improve the file.

Reply:
Yes just do that please.